### ISSUE TITLE
Ok, installed and ran sysmon ... Now what?

### ISSUE THREAD

**Issue by quantuumsnot** [SwiftOnSecurity/sysmon-config#114]
For the average user it's not clear enough from `Use` section about how to check the results of sysmon in `Event Viewer` for possible malicious activity on the machine

**Comment by davebremer** [comment]
Yes analysis is hard and not for the “average user”. Says on collects data. It does not analyse. While a SIEM is pretty much essential given the volume of events. It takes knowledge and experience that needs developed in a person over time. An average user won’t have much luck, unless they work at developing those skills, but then that’s not the average user

**Comment by quantuumsnot** [comment]
Average user != Beginner which usually means a simple how-to will be sufficient enough
